pyproject.toml

```
@@ -0,0 +1,65 @@
+[build-system]
+requires = ["setuptools", "wheel", "versioneer"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "MembraneCurvature"
+version = "1.0.0"
+license = {file = "LICENSE"}
+description = "MDAnalysis tool to calculate membrane curvature from MD simulations."
+authors = [
+    {name = "Estefania Barreto-Ojeda", email = "estefania@ojeda-e.com"},
+]
+maintainers = [
+    {name = 'MDAnalysis Core Developers', email = 'mdanalysis@numfocus.org'}
+]
+readme = "README.md"
+requires-python = ">=3.9"
+keywords = [
+    "python", "biophysics", "molecular-dynamics",
+    "molecular-simulation", "analysis",
+    "trajectory-analysis",
+]
+dependencies = [
+    'numpy>=1.20.0',
+    'mdanalysis>=2.0.0'
+    ]
+
+classifiers = [
+        'Development Status :: 5 - Production/Stable',
+        'Environment :: Console',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
```

```suggestion
        'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',
```
Apologies, I can't remember if this repo is GPLv3+ or GPLv3 pure

---

Thanks for checking this, it's GPLv3 (with no plus). 100% positive.